src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,444 @@
+
+{{ MACROS___make_filetree_example(
+   {
+   "sub-01": {
+      "micr": {
+         "sub-01_sample-01_SEM.png": "",
+         "sub-01_sample-02_SEM.png": "",
+         "sub-01_sample-03_SEM.png": "",
+         "sub-01_SEM.json": "",
+         },
+      }
+   }
+) }}
+
+In this example, the JSON metadata is common for all samples of `sub-01`.
+JSON metadata may be defined per subject or per sample as appropriate, as per the
+[inheritance principle](../02-common-principles.md#the-inheritance-principle).
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity corresponds to a custom label that
+MAY be used to distinguish a different set of parameters used for acquiring the same modality.
+For example, two images of the same sample acquired by bright-field microscopy (BF) in PNG format at
+different magnification of 40x and 60x.
+In such case two files could have the following names: `sub-01_sample-01_acq-40x_BF.png` and
+`sub-01_sample-01_acq-60x_BF.png`, however the user is free to choose any other label as long as
+they are consistent across subjects and sessions.
+
+The [`stain-<label>`](../99-appendices/09-entities.md#stain) entity MAY be used to distinguish
+image files from the same sample using different stains or antibodies for contrast enhancement.
+
+For example: One brain slice (`sample-01`) extracted from subject `sub-01` with three
+stains (`stain-01`, `stain-02` and `stain-03`) in three separate files, imaged by selective plane
+illumination microscopy (SPIM) in OME-TIFF format
+
+{{ MACROS___make_filetree_example(
+   {
+   "sub-01": {
+      "micr": {
+         "sub-01_sample-01_stain-01_SPIM.ome.tif": "",
+         "sub-01_sample-01_stain-01_SPIM.json": "",
+         "sub-01_sample-01_stain-02_SPIM.ome.tif": "",
+         "sub-01_sample-01_stain-02_SPIM.json": "",
+         "sub-01_sample-01_stain-03_SPIM.ome.tif": "",
+         "sub-01_sample-01_stain-03_SPIM.json": "",
+         },
+      }
+   }
+) }}
+
+In this example, the entity stain is used to distinguish images with different
+stains in separate files from the same sample.
+In the case where a single file contains different staining in each channel, the
+`stain-<label>` is omitted.
+
+Stains SHOULD be indicated in the `"SampleStaining"` key in the sidecar JSON file,
+although the label may be different.
+Description of antibodies SHOULD also be indicated in `"SamplePrimaryAntibodies"`
+and/or `"SampleSecondaryAntobodies"` as appropriate.
+
+If more than one run of the same sample, acquisition and stain are acquired during the same
+session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used:
+`_run-1`, `_run-2`, `_run-3`, and so on.
+If only one run was acquired the `run-<index>` can be omitted.
+
+The [`chunk-<index>`](../99-appendices/09-entities.md#chunk) entity is used when multiples
+regions (2D images or 3D volumes files) of the same physical sample are imaged with different
+fields of view, regardless if they overlap or not.
+
```

Comment:
```suggestion
The term “chunk” is similar to others, such as, “slice”, “slab” and “patch”, and was chosen as best representing the concept of a image or volume “portion”. 
```

Maybe is good to mention other terms that are similar or equivalent, so people that use them may recognize it?

Reply:
I understand your point but I'm afraid that adding new terms that are not part of the specification will be more confusing for people who are not used to them.

I would suggest to add the word "portion" to the original sentence and change the word "multiple" for different", ex:
```
The `chunk-<index>` entity is used when different regions/portions (2D image or 3D volume files) of the same physical sample are imaged with different fields of view, regardless if they overlap or not.
```